package.json

```diff
@@ -144,7 +144,7 @@
     "@reduxjs/toolkit": "1.9.0",
     "@sentry/react": "7.22.0",
     "@sentry/tracing": "7.22.0",
-    "@voltz-protocol/v1-sdk": "1.3.1",
+    "@voltz-protocol/v1-sdk": "1.4.0",
```

Comment:
I have requested figma access to the design so I can inspect and review properly. Currently I don't have access @ssbarbee

Reply:
Hey @Sol3iman i dont own the figmas. Can you reach out to Brokoli or Renato? I am not working in the next few days so please reach out to them 🙏